package.json

```diff
@@ -64,7 +64,7 @@
   "dependencies": {
     "@hapi/crumb": "^8.0.0",
     "@hapi/hoek": "^9.0.4",
-    "@vrbo/steerage": "^12.1.3",
+    "@vrbo/steerage": "git+https://github.com/ExpediaGroup/steerage.git#8faae0165cfd325f75c954df2068cac8a0baae08",
```

Comment:
This is a source reference; not a version

Reply:
yeah! i was playing with... and testing!

Closing this pr!